packages/graphback-runtime-mongodb/src/queryBuilder.ts

```diff
@@ -75,6 +75,24 @@ function isPrimitive(test: any): boolean {
 };
 
 
+function dangerousHack(filter: any, key: string): any {
```

Comment:
What is this hack doing? Maybe we can name it that way and have a comment about the hack and what's needed to get rid of the hack in the future - e.g a link to an issue / PR etc.

Reply:
Thanks! Please see #1349 I believe it covers this, plus I have added some comments in the same file:
https://github.com/aerogear/graphback/blob/4fd1c80b7f0126e366924ab8b7c1ddde7e8bcbad/packages/graphback-runtime-mongodb/src/queryBuilder.ts#L77-L84

Reply:
This is great. Nice work.